particles/fcc/particle.py

```diff
@@ -22,7 +24,7 @@ def __init__(self, fccobj):
         self._tlv = TLorentzVector()
         p4 = fccobj.core().p4
         self._tlv.SetXYZM(p4.px, p4.py, p4.pz, p4.mass)
-        #write(str('Made Pythia {}').format(self))
+        self._uid = Identifier.make_id(Identifier.PFOBJECTTYPE.PARTICLE, fccobj.getObjectID().index, subtype, self._tlv.E())
```

Comment:
Did you make sure that if the particle is not an fcc.Particle, _uid is properly defined?

Reply:
No, I did the opposite and allowed _uid to be undefined and that the code works that we. We should discuss!